includes/modules/pages/checkout_confirmation/jscript_paypal_incontext.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Javascript enabler for providing and handling the PayPal 
+ *   Express In Context shopping experience.
+ *
+ * @package page
+ * @copyright Copyright 2003-2016 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Author: mc12345678  Sat Aug 13 11:35:35 2016 -0400 Modified in v1.6.0 $
+ */
+ 
+  if (   defined('MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE') && MODULE_PAYMENT_PAYPALWPP_CHECKOUTSTYLE == 'InContext' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_MERCHANTID')    && MODULE_PAYMENT_PAYPALWPP_MERCHANTID    != '' 
+      && defined('MODULE_PAYMENT_PAYPALWPP_STATUS')        && MODULE_PAYMENT_PAYPALWPP_STATUS        == 'True') {
+        
+//        require(DIR_WS_MODULES . 'payment/paypal/paypalwpp_incontext.php');
+if (isset($_SESSION['paypal_ec_token']) && !empty($_SESSION['paypal_ec_token'])
+    && isset($_SESSION['paypal_ec_markflow']) && $_SESSION['paypal_ec_markflow'] = 1) {
+
+}?>
```

Comment:
Non-essential if the overall concept/operation is accepted, something of an alternate path that would need to be used/taken advantage of if the provided method of "clearing" the paypal_ec_token (left the paypal_ec_markflow unchanged) is considered undesirable/unacceptable when ec_cancel=1 is a parameter of the url. 

Is primarily a remnant that can be removed for operational sake, but the interaction of all these parts is pretty extensive.

The vital part if all else is accepted is the $buttonContainer and require of the file. Otherwise thus far haven't found a way (outside of a potential init_ file or other file loaded on page load/redirect) that would allow accomplishing the same functionality already offered by ZC when pressing the confirm order button while a token is assigned to an "active" connection.